Sdk/Track/Swrve.m

```diff
@@ -731,7 +731,7 @@ -(id) initWithAppID:(int)swrveAppID apiKey:(NSString*)swrveAPIKey config:(SwrveC
 
         // If this is the first time this user has been seen send install analytics
         if(didSetUserId) {
-            [self event:@"Swrve.first_session"];
+            [self eventInternal:@"Swrve.first_session" payload:nil triggerCallback:true];
```

Comment:
It was calling the regular event method which had a default of true for triggerCallback.